src/Controllers/SchemaController.js

```diff
@@ -1405,6 +1441,8 @@ export default class SchemaController {
       }
       // requiresAuthentication passed, just move forward
```

Comment:
Stumbled upon this old comment I though `authenticated` is a better description of a group of users than `requiresAuthentication`, just left the former variant for backwards compatibility. I'd propose to replace it in all the docs and may be drop completely by next major release or whenever. This also can be applied to CLP.